src/blocking_impl.rs

```diff
@@ -157,12 +157,11 @@ mod tests {
         let end = datetime!(2020-01-31 23:59:59.99 UTC);
 
         let resp = provider.get_quote_history("AAPL", start, end);
-        if resp.is_ok() {
```

Comment:
Random but while looking at the tests, also added this change.
`resp.is_okay()` should be an assert I assume.
for example, if you used these tests with the dates 1969-01-01, this test would pass, even though the response is an error (since it doesn't support pre-epoch time currently)

Happy to remove this if someone disagrees, but seems logical to me.